pom.xml

```diff
@@ -42,6 +42,8 @@
 
     <properties>
         <jenkins.version>1.580.1</jenkins.version>
+        <java.level>6</java.level>
+        <findbugs.failOnError>false</findbugs.failOnError>
```

Comment:
This was the setting in the old pom; I kept it just in case there was some issue that was known.  Since there are no failing tests, I don't think it'd be a problem to change it to `true`.  Unless there's pushback from the plugin maintainers?